lib/modules/platform/gitea/index.md

```diff
@@ -1,25 +1,38 @@
 # Gitea and Forgejo
 
-Renovate supports [Gitea](https://gitea.io) and the new fork [Forgejo](https://forgejo.org).
+Renovate supports [Gitea](https://gitea.io) and the [Forgejo](https://forgejo.org) fork.
 Forgejo exists since `v1.18.0` and is currently fully compatible to Gitea.
 
 ## Authentication
 
-First, [create a Personal Access Token](https://docs.gitea.io/en-us/api-usage/#authentication) for the bot account.
-Let Renovate use your PAT by doing _one_ of the following:
+First, [create a Personal Access Token (PAT)](https://docs.gitea.io/en-us/api-usage/#authentication) for the bot account.
+Then let Renovate use your PAT by doing _one_ of the following:
 
 - Set your PAT as a `token` in your `config.js` file
 - Set your PAT as an environment variable `RENOVATE_TOKEN`
 - Set your PAT when you run Renovate in the CLI with `--token=`
 
-Remember to set `platform=gitea` somewhere in your Renovate config file.
-Either the account should have full name and email address set to allow Renovate to estabilish Git identity, or the `gitAuthor` config option should be set.
+You must set `platform=gitea` in your Renovate config file.
+
+Either the account should have full name and email address set to allow Renovate to establish Git identity, or the `gitAuthor` config option should be set.
+
+### If you do not set `username` or `gitAuthor`
+
+If you don't set `username` or `gitAuthor`, then you must use the user scope.
+User scopes need Gitea `v1.19` or higher.
+
+The user scope must have these permissions:
+
+- `repo`
```

Comment:
this is always needed, it's not a user scope 

https://blog.gitea.io/2023/03/gitea-1.19.0-is-released/#-restrict-access-token-permissions-20908httpsgithubcomgo-giteagiteaissues20908